src/backend/common/DefaultMemoryManager.cpp

```diff
@@ -206,8 +208,8 @@ void *DefaultMemoryManager::alloc(bool user_lock, const unsigned ndims,
 size_t DefaultMemoryManager::allocated(void *ptr) {
     if (!ptr) return 0;
     memory_info &current = this->getCurrentMemoryInfo();
-    locked_iter iter     = current.locked_map.find((void *)ptr);
-    if (iter == current.locked_map.end()) return 0;
+    auto iter            = current.locked_map.find(ptr);
```

Comment:
nit: I am aware this variable is not used in a very large scope, but it could also use a better name to help while reading the code.